ember-cli-build.js

```diff
@@ -62,6 +62,11 @@ module.exports = function () {
           { removeViewBox: false },
           { removeTitle: false },
           { removeDesc: false },
+          {
+            removeUnknownsAndDefaults: {
+              unknownContent: false,
```

Comment:
svgo was removing drop shadows from one of the new SVGs. This change prevents that. Not 100% sure it's worth it though